### ISSUE TITLE
Noto PRs which need attention

### ISSUE THREAD

**Issue by RosaWagner** [google/fonts#6671]
After reviewing all the Noto PRs, this is the list of the ones that need attention:

## Open PRs:

- [x] https://github.com/google/fonts/pull/5308
- [x] https://github.com/google/fonts/pull/6401
- [x] https://github.com/google/fonts/pull/6407
- [x] https://github.com/google/fonts/pull/5758

## Merged PRs:

- [ ] https://github.com/google/fonts/pull/6695
- [ ] https://github.com/google/fonts/pull/6110
- [x] https://github.com/google/fonts/pull/4654
- [x] https://github.com/google/fonts/pull/4653
- [x] https://github.com/google/fonts/pull/4655
- [x] https://github.com/google/fonts/pull/4657
- [x] https://github.com/google/fonts/pull/5535
- [x] https://github.com/google/fonts/pull/6691
- [x] https://github.com/google/fonts/pull/6694
- [x] https://github.com/google/fonts/pull/6692
- [x] https://github.com/google/fonts/pull/6691
- [x] https://github.com/google/fonts/pull/5287
- [x] https://github.com/google/fonts/pull/6432
- [x] https://github.com/google/fonts/pull/6426
- [x] https://github.com/google/fonts/pull/6427
- [x] https://github.com/google/fonts/pull/5281
- [x] https://github.com/google/fonts/pull/5283
- [x] https://github.com/google/fonts/pull/5278
- [x] https://github.com/google/fonts/pull/6612
- [x] https://github.com/google/fonts/pull/6418
- [x] https://github.com/google/fonts/pull/6419
- [x] https://github.com/google/fonts/pull/6469
- [x] https://github.com/google/fonts/pull/5369
- [x] https://github.com/google/fonts/pull/5377
- [x] https://github.com/google/fonts/pull/5395

**Comment by simoncozens** [comment]
For CJK we just need to keep #6695, fixing that will fix the rest.

**Comment by emmamarichal** [comment]
@simoncozens Ok thanks! so I can remove all of this from the traffic jam board, and stop to track them right?
https://github.com/google/fonts/pull/4654
 https://github.com/google/fonts/pull/4655
 https://github.com/google/fonts/pull/4657
 https://github.com/google/fonts/pull/5535
 https://github.com/google/fonts/pull/6691
 https://github.com/google/fonts/pull/6694
 https://github.com/google/fonts/pull/6692
 https://github.com/google/fonts/pull/6691

And what about Oriya and Lycian?

**Comment by simoncozens** [comment]
Yes please, all those can go. I am still working on Oriya, and Lycian needs a glyphset update, blocked on @garretrieger.

**Comment by garretrieger** [comment]
Just submitted the lycian change internally. Should be available on sandbox in a day. As usual affected families will need to be repushed.

**Comment by emmamarichal** [comment]
@garretrieger @simoncozens 
Ok great, thanks!
Then I delete the Lycian PR from the project board. @simoncozens don't hesitate to assign me to the review of the new PR

**Comment by simoncozens** [comment]
Sure, I think this issue is now up-to-date and can be closed once #7347 is merged.
